### PR TITLE
clusterautoscaler helm chart fixes for clusterAPI cloud provider

### PR DESCRIPTION
**Pull request by ugiraud**
#### Which component this PR applies to?

cluster-autoscaler

#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

This PR fixes :
  * Role creation for cluster autoscaler when in mode kubeconfig-incluster
  * Kubeconfig secret volume and volumemount wasn't present when using clusterAPIKubeconfigSecret value
  * Namespace wasn't correctly set when in mode kubeconfig-incluster

#### Does this PR introduce a user-facing change?

```release-note
New property in values : clusterAPIConfigMapsNamespace to allow setting workload cluster namespace to store cluster autoscaler status configmap.
```

### CODE REVIEW

**Review by gjtempleton — COMMENTED** [review]
Thanks for this, one minor nitpick, but other than that, this looks good to me.

File: charts/cluster-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.15.0
+version: 9.15.1
```
[the remark this answers is not in the document]
Comment (ugiraud) [reply charts/cluster-autoscaler/Chart.yaml]: Yes, I've just bumped minor version.

**Comment by gjtempleton** [comment]
Thanks for that. One more minor thing, can you squash the commits down to 4 (i.e. squash the two version bumps together), then this looks good to go.

**Comment by ugiraud** [comment]
> Thanks for that. One more minor thing, can you squash the commits down to 4 (i.e. squash the two version bumps together), then this looks good to go.

Done, the last 2 have been squashed

**Comment by gjtempleton** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**